src/expr.cpp

```diff
@@ -1307,7 +1307,7 @@ UnaryExpr *UnaryExpr::Instantiate(TemplateInstantiation &templInst) const {
 
 void UnaryExpr::Print(Indent &indent) const {
     if (!expr || !GetType()) {
-        indent.Print("UnaryExpr: <NULL EXPR>\n");
```

Comment:
It looks like a reference to NULL value of ISPC language, isn't it?

Reply:
It refers to invalid expression (NULL pointer instead of an expression itself or invalid type).

I'm not sure if we should change it or not, seems that keeping at as is is good option.

Reply:
I've updated the PR and summarized what I did in "update" section of initial description.